src/services/twilio-api/twilio_preview.json

```diff
@@ -2597,6 +2597,309 @@
             "nullable": true
           }
         }
+      },
```

Comment:
This file need not be changed since these spec files will be overwritten by released spec files from oai. If your changes are there in open-api or api-definitions, they will be coming here as well. No need to do it explicitly here.